accessclinicaltrials.niaid.nih.gov/portal/gitops.json

```diff
@@ -1,5 +1,5 @@
 {
-  "gaTrackingId": "UA-119127212-1",
```

Comment:
Is this the tracking ID for Google Analytics, if so, it may be intentional that prod and QA are different.

Reply:
oh yes, we should use `-1`